tests/test_special_euclidean.py

```diff
@@ -1100,8 +1100,8 @@ def test_log_then_exp_left_with_angles_close_to_pi(self):
                     inv_expected = helper.to_vector(inv_expected)
 
                     self.assertTrue(
-                        gs.allclose(result, expected)
-                        or gs.allclose(result, inv_expected))
+                        gs.allclose(result, expected, atol=1e-6)
```

Comment:
Define constant variable ATOL=1e-6 at the top of the file? It could be easier to change it later.

Reply:
For some tests, I needed to increase it more than that (up to `5e-3` for angles close to pi, with Tensorflow and Pytorch). Leaving it like that might allow later improvements of the functions ?
We might be able to get away with the rtol though I'll check that.